### DISCUSSION TITLE
Storage: Git to self hosted git system

### DISCUSSION THREAD

**Discussion by k1ngf15h3r** [requarks/wiki#5008] [Help / Questions] [answered]
Hey Community,

my search does not give me the right answer or maybe I searched wrong but I have a problem connecting my really new wiki.js (Wiki.js 2.5.275) to my private hosted gitea server. I run wiki.js in a docker-compose environment on a ubuntu root server.

First of all, is it possible to connect to a self hosted git instance? Hope it is, because all my projects are in this self hosted git.

Glad to read your answers.

**Comment by NGPixel** [comment, marked as answer]
As long as your git server is accessible from the Wiki.js server, then yes.
